sos/report/plugins/filesys.py

```diff
@@ -35,6 +35,21 @@ class Filesys(Plugin, DebianPlugin, UbuntuPlugin, CosPlugin):
     ]
 
     def setup(self):
+        self.add_forbidden_path([
+            # cifs plugin
+            '/proc/fs/cifs',
+            # lustre plugin
+            '/proc/fs/ldiskfs',
+            # nfs plugin
+            '/proc/fs/nfsd',
+            '/proc/fs/nfsfs',
```

Comment:
Ditto, but again, since nfs doesnt care about other files than `/proc/fs/nfsfs/servers` and `/proc/fs/nfsfs/volumes`, no need to collect them here either.